meilisearch-http/tests/content_type.rs

```diff
@@ -0,0 +1,109 @@
+mod common;
+
+use crate::common::{GetAllDocumentsOptions, Server};
+use actix_web::test;
+use meilisearch_http::create_app;
+use serde_json::{json, Value};
+
+#[actix_rt::test]
+async fn strict_json_bad_content_type() {
+    let routes = [
+        // all the POST route except the dumps that can be created without any body or content-type
```

Comment:
```suggestion
        // all the POST routes except the dumps that cannot be created without any body or content-type
```

Reply:
"Cannot" right? Or maybe I missed something

Reply:
Also the PUT methods, I don't know if you can add them in this test
- update documents 
- update index (I forgot this one in the issue, I've just updated it)

Reply:
I duplicated all the tests for the index document route with the put verb :+1: